src/apps/ipv4/fragment.lua

```diff
@@ -89,6 +92,23 @@ function Fragmenter:new(conf)
    assert(o.mtu >= 68)
    o.next_fragment_id = deterministic_first_fragment_id or
       math.random(0, 0xffff)
+
+   alarms.add_to_inventory {
+      [{alarm_type_id='outgoing-ipv4-fragments'}] = {
+         resource=tostring(S.getpid()),
+         has_clear=true,
+         description='Outgoing IPv4 fragments over N fragments/s',
+      }
+   }
+   local outgoing_fragments_alarm = alarms.declare_alarm {
+      [{resource=tostring(S.getpid()),alarm_type_id='outgoing-ipv4-fragments'}] = {
+         perceived_severity='warning',
+         alarm_text='More than 10,000 outgoing IPv4 fragments per second',
+      }
+   }
```

Comment:
I think in some future there's room for some refactoring perhaps; probably joining these three things (the inventory addition, the specific alarm, and the CounterAlarm).